src/main/java/uk/gov/hmcts/reform/professionalapi/controller/internal/OrganisationInternalController.java

```diff
@@ -234,22 +285,35 @@ public ResponseEntity updatesOrganisation(
     @ApiResponses({
             @ApiResponse(
                     code = 201,
-                    message = "User has been added",
+                    message = "The new User has been added to the Organisation",
                     response = OrganisationResponse.class

```

Comment:
shouldn't it be NewUserResponse.java

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-professional-api/pull/503/commits/78ec7e12704dde1f85bb4ca01c405ef6b5449462